src/main/java/seedu/contax/model/person/Person.java

```diff
@@ -82,6 +82,10 @@ public Person withoutTag(Tag tag) {
         return new Person(name, phone, email, address, updatedTag);
     }
 
+    public boolean hasTag(Tag tag) {
```

Comment:
Missing Javadocs, though, I think this is fairly self-explanatory.

Reply:
Fixed in 7ff3cf68